src/BSeries.jl

```diff
@@ -1,5 +1,5 @@
 module BSeries
-
+    include("Energy_Preserving.jl")
```

Comment:
Please move this below - somewhere where it fits better. For example, you may want to dispatch on things - that need to be defined first

Reply:
@ranocha Sorry, @Sondar74 and I have no idea where it would fit better.  We had to hunt for a while to figure out the syntax, and the examples we found put includes at the top.  I also don't understand the comment "you may want to dispatch on things" -- can you clarify?

Reply:
Say you want to have a simplified API such as `is_energy_preserving(rk::RungeKuttaMethod, order)`. To define such a method dispatching on its first argument, `RungeKuttaMethod` must be defined - which comes from RootedTrees.jl. Thus, the definition of this method must come after `using RootedTrees`.

Reply:
A good place may just be at the end of the main file - near the line 
```julia
include("latexify.jl")
```

There, everything you may need for dispatch should be defined.